internal/store/postgres/store.go

```diff
@@ -41,6 +41,15 @@ func NewStore(c *store.Config) (*Store, error) {
 		log.Panic(err)
 	}
 
+	db, err := gormDB.DB()
+	if err != nil {
+		log.Panic(err)
+	}
+	db.SetConnMaxLifetime(c.ConnMaxLifeTime)
+	db.SetMaxIdleConns(c.MaxIdleConns)
+	db.SetMaxOpenConns(c.MaxOpenConns)
+	db.SetConnMaxIdleTime(c.MaxIdleLifeTime)
```

Comment:
let's add conditions to check if each of these configs doesn't have empty/nil value only then we will configure

Reply:
@rahmatrhd Is a check needed? Wouldn't the default values configured automatically apply here?

Reply:
I was thinking if user doesn't explicitly define these additional the config, then let's also not set their values. With this we can also skip adding default values.

Reply:
It's generally a good practice to set some default DB connection parameters, which the user can override later if they need to. Not setting any is not a great idea. 😅

Reply:
I see, I think we can keep it then 👍